install-shipwright.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+#
+# Installs Shipwright Build Controller and Build-Strategies using the Makefile target.
+#
+
+set -eu -o pipefail
+
+source common.sh
+
+readonly REPO_NAME="build"
+CLONE_DIR="."
+
+if ! is_current_repo "${REPO_NAME}" ; then
+	CLONE_DIR="${GITHUB_WORKSPACE}/src/${REPO_NAME}"
+fi
+
+cd "${CLONE_DIR}" || fail "Directory '${CLONE_DIR}' does not exit!"
+
+echo "# Deploying Shipwright Controller (${CLONE_DIR})..."
+make install-controller-kind
```

Comment:
This is what I think will not work if the cluster name is not called `kind`.

Reply:
Now I understand where [you're coming from](https://github.com/google/ko/blob/bd8cfaa245ab6c3e35b8073e97b67aaa6941419b/README.md?plain=1#L268-L271), @SaschaSchwarze0. Well, I just tested with a cluster named `cruel` (instead of the default `kind`) and it works... so maybe you can check that as well and share what you find, please.

Reply:
Sorry, @SaschaSchwarze0, I end up making those changes on my personal project 🤦‍♀️ In fact, on using a different cluster name it fails as you've pointed before.

Reply:
Please consider the latest changes.